internal/kafka/internal/workers/clusters_mgr.go

```diff
@@ -38,7 +38,7 @@ import (
 
 const (
 	observabilityNamespace          = "managed-application-services-observability"
-	observabilityCatalogSourceImage = "quay.io/rhoas/observability-operator-index:v3.0.9"
+	observabilityCatalogSourceImage = "quay.io/rhoas/observability-operator-index:v3.0.10"
```

Comment:
maybe its time we make this configurable via cli flags

Reply:
I can make a Jira to make this change. Thanks @machi1990

Reply:
That will be perfect, thanks @vmanley

Reply:
Jira to make OO version configurable: https://issues.redhat.com/browse/MGDSTRM-8588

Reply:
Thanks